packages/react-jss/package.json

```diff
@@ -33,7 +33,8 @@
     "decorator"
   ],
   "scripts": {
-    "prepare": "node ../../scripts/prepare.js"
+	"prepare": "node ../../scripts/prepare.js",
+	"flow": "flow"
```

Comment:
Again, we don't need to run flow here as it's being run from the root directory

Reply:
Removed